### ISSUE TITLE
Feature Request: Add the 'magic barcodes' to the other tabs

### ISSUE THREAD

**Issue by kennioje** [CompositeLight/currentRMS-helper#78]
Add the use of the 'magic barcodes' (e.g. 'Container', 'revert', etc.) to the Prepare, Book out and Check-in tabs. Not all are relevant everywhere, but I often do setting of containers in book out (I often set the vehicle as a container, to get a guide as to weight on the van).

**Comment by CompositeLight** [comment]
Good point. Hadn't thought of use cases for the other tabs... I've made container, revert and remove work in the Book Out tab for 1.31.21, currently live on the dev branch.

Extra note: You can also now use the Booked Out Weight display for mutli-van loading purposes (though I see the logic in your container approach, where you can't load them in consecutively. Good idea!).
